services/realtimeclock.md

```diff
@@ -0,0 +1,33 @@
+# Real time clock
+
+    
+    identifier: 0x1a8b1a28
+    extends: _sensor
+
+## Registers
```

Comment:
we could also have error register, which counts the time since last `set_time` and based on precisions computes possible error in seconds

Reply:
would that be for low quality clocks?

Reply:
even the high-precision one from adafruit https://www.adafruit.com/product/3013 is 2ppm which is 5s/month; the not so good ones are 2s/day